includes/templates/class-amp-post-template.php

```diff
@@ -342,8 +342,14 @@ private function build_post_comments_data() {
 	 * Build post content.
 	 */
 	private function build_post_content() {
-		/** This filter is documented in wp-includes/post-template.php */
-		$content = apply_filters( 'the_content', get_the_content( null, false, $this->post ) );
+		if ( post_password_required( $this->post ) ) {
+			$content = get_the_password_form( $this->post );
+		} else {
+			// Note: This is intentionally not using get_the_content() because the legacy behavior of posts in Reader
+			// mode is to display multi-page posts as a single page without any pagination links.
+			/** This filter is documented in wp-includes/post-template.php */
```

Comment:
Attaching two different comments together seems weird here. How about this:
```suggestion
			/**
			 * This filter is documented in wp-includes/post-template.php.
			 * 
			 * Note: This is intentionally not using get_the_content() because the legacy behavior of posts in
			 * Reader mode is to display multi-page posts as a single page without any pagination links.
			 */
```

Reply:
I originally intended it to be separate, as the inline comment is talking about the parameter and the PhpDoc comment is talking about the filter as a whole. But I'm fine with combining.